bindata/assets/alerts/kube-apiserver-slos.yaml

```diff
@@ -303,111 +303,111 @@ spec:
         (
           (
             # too slow
-            sum(rate(apiserver_request_duration_seconds_count{job="apiserver",verb=~"POST|PUT|PATCH|DELETE"}[1d]))
+            sum(rate(apiserver_request_duration_seconds_count{job="apiserver",verb=~"POST|PUT|PATCH|DELETE",subresource!~"proxy|log|exec|portforward"}[1d]))
```

Comment:
it looks like we are missing `attach` subresource

Reply:
additionally this is what we have defined for OpenShift (note these are endpoints not resources)

```
// request paths that match this regular expression will be treated as long running
// and not subjected to the default server timeout.
const originLongRunningEndpointsRE = "(/|^)(buildconfigs/.*/instantiatebinary|imagestreamimports)$"
```

Reply:
I think we can postpone those, and focus only on subresources in this PR, this rules are complex and is easy to go wrong and it will be difficult to troubleshoot problems if we do so many changes at once

Reply:
added attach subresource